pkg/operators/wasm/testdata/Makefile

```diff
@@ -0,0 +1,29 @@
+ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+
+TEST_ARTIFACTS = \
+	fields \
+	dataarray \
+	badguest \
+	#
+
+all: $(TEST_ARTIFACTS)
+
+.PHONY: $(TEST_ARTIFACTS)
+$(TEST_ARTIFACTS):
+	@echo "Building $@"
+	@sudo -E IG_EXPERIMENTAL=true \
+		IG_SOURCE_PATH=$(realpath $(ROOT_DIR)/../../../..) \
+		ig image build \
+		-t $@:latest \
+		$@
+	@sudo -E IG_EXPERIMENTAL=true \
+		ig image export $@:latest $@.tar
+	# This fails with Error: removing gadget image: unable to reload index
+	#@sudo -E IG_EXPERIMENTAL=true \
+	#	ig image remove $@:latest
```

Comment:
What is the reason for this error?

Does `ig` has some flag to change the oci store directory? If so, we could use `mktemp` + `ig image build --oci-store=/tmp/...`.

Reply:
I checked and it doesn't happen anymore, no idea what was going on.

Reply:
Ah, it's back. I'll open an issue about it after merging the PR.